dist/logger/serverLogger.js

```diff
@@ -77,11 +77,12 @@ module.exports = function (app, _ref) {
         onMessage(event);
```

Comment:
The package.json build scripts don't seem to output to this folder?

Reply:
Corrected in 2cd0841964a740a577946d6ace9dbe804fed87a5